### DISCUSSION TITLE
Is "INSERT OR IGNORE" implemented in the emulator?

### DISCUSSION THREAD

**Discussion by qshen2000** [GoogleCloudPlatform/cloud-spanner-emulator#168] [Q&A]
It seems not working on my local emulator. Got this: "ERROR: spanner: code = "AlreadyExists","

**Comment by skuruppu** [comment]
The feature should work on `v1.5.17`. See https://github.com/GoogleCloudPlatform/cloud-spanner-emulator/issues/169.
